cluster/node-pools/master-default/userdata.yaml

```diff
@@ -767,6 +767,14 @@ write_files:
             - group: "" # core
               resources: ["pods"]
         {{ end }}
+{{- eq .Cluster.ConfigItems.audit_kube_controller_manager_node_changes "true"}}
+        - level: Request
+          users: ["system:kube-controller-manager"]
+          verbs: ["patch", "update"]
+          resources:
+            - group: "" # core
+              resources: ["nodes"]
+{{- end}}
```

Comment:
I think the `{{-` will bite you in your yaml

Reply:
`{[-` will trim whitespace such that the line is not there after rendering.

However, I forgot `if` keyword above so e2e failed :facepalm:

Reply:
ah oh

Sometime the whitespace cleanup does too much but yeah that was clearly the error